Ocean-back/Handlers/QuizQuestionHandlers/GetQuestionHandler.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Marten;
+using MediatR;
+using Ocean_back.Model;
+
+namespace Ocean_back.Handlers.QuizQuestionHandlers
+{
+    public class GetQuestion
+    {
+        public class Request : IRequest<Response>
+        {
+            public Guid CharId { get; set; }
+        }
+        public class Response
+        {
+            public Guid CharId { get; set; }
+            public string Description { get; set; }
+            public List<Characteristic> Characteristics { get; set; }
+        }
+        public class Handler : IRequestHandler<Request, Response>
+        {
+            private readonly IDocumentSession _session;
+            public Handler(IDocumentSession session) => _session = session;
+            public async Task<Response> Handle(Request request, CancellationToken cancellationToken)
+            {
+                var question = await _session.LoadAsync<QuizQuestion>(request.CharId);
+                if (question == null)
+                    throw new Exception("Quiz question was not found");
+
+                return new Response
+                {
+                    CharId = question.Id,
```

Comment:
QuestionId instead of CharId

Reply:
![kjamil_response](https://github.com/WebZaLeb/OCEAN-WebApp/assets/46653449/a9118aa3-cfd2-49c8-9d83-e37206c04fe0)